### PR TITLE
913 ebt card in office

### PR DESCRIPTION
**Pull request by IlseFerris**

### CODE REVIEW

File: notes/expedited-screening.vbs [hunk 1]
```diff
@@ -147,54 +152,101 @@ If (int(income) < 150 and int(assets) <= 100) or ((int(income) + int(assets)) <
...
-Call navigate_to_MAXIS_screen("STAT", "DISQ")
-Call MAXIS_footer_finder(MAXIS_footer_month, MAXIS_footer_year) 'grabbing footer month and year
-
-'Reads the DISQ info for the case note.
-EMReadScreen DISQ_member_check, 34, 24, 2
-If DISQ_member_check = "DISQ DOES NOT EXIST FOR ANY MEMBER" then
-	has_DISQ = False
+'Output if case is priv and user selects to run the script.
+If priv_choice = vbYes then
+    closing_message = ("This result of the expedited screening is: " & expedited_status & ".")
```
Comment (C-Love) [inline notes/expedited-screening.vbs:157]: Add information here that case is PRIV and no NOTE has been made.

File: notes/expedited-screening.vbs [hunk 2]
```diff
@@ -147,54 +152,101 @@ If (int(income) < 150 and int(assets) <= 100) or ((int(income) + int(assets)) <
...
+    	call write_variable_in_CASE_NOTE("         CAF 1 liquid assets claimed: $" & assets)
+    	call write_variable_in_CASE_NOTE("         CAF 1 rent/mortgage claimed: $" & rent)
+    	call write_variable_in_CASE_NOTE("        Utilities (amt/HEST claimed): $" & utilities)
+    	call write_variable_in_CASE_NOTE("---")
+    	If has_DISQ = True then call write_variable_in_CASE_NOTE("A DISQ panel exists for someone on this case.")
+    	If has_DISQ = False then call write_variable_in_CASE_NOTE("No DISQ panels were found for this case.")
+        If in_office_card = False then Call write_variable_in_CASE_NOTE("Recipient will NOT be able to get an EBT card in an agency office. An EBT card has previously been provided to the household.")
+    	call write_variable_in_CASE_NOTE("---")
+    	call write_variable_in_CASE_NOTE(worker_signature)
+    	If expedited_status = "client appears expedited" then closing_message = "This client appears expedited. A same day interview needs to be offered."
```
Comment (C-Love) [inline notes/expedited-screening.vbs:204]: Add details to the closing message that the NOTE was made.

File: notes/expedited-screening.vbs [hunk 3]
```diff
@@ -58,20 +59,26 @@ changelog_display
...
-    If snap_pend_check = "PEND" Then
-        EMReadScreen snap_app_date, 8, 10, 33
-        application_date = replace(snap_app_date, " ", "/")
-    End If
-    transmit
-    EMReadScreen check_for_hcre, 4, 2, 50
-    If check_for_hcre = "HCRE" Then
-        PF10
-    End If
+    Call navigate_to_MAXIS_screen_review_PRIV("STAT", "PROG", is_this_priv)
```
Comment (C-Love) [inline notes/expedited-screening.vbs:66]: Even though we allow this to continue, we shouldn't do this until after the dialog. We have had issues where the wrong case number is pulled and the worker should have a chance to check the case number before we just don't even try to case note.

**Review by C-Love — CHANGES_REQUESTED** [review]
A couple quick changes

File: notes/expedited-screening.vbs (hunk 3 above)
Comment (IlseFerris) [reply notes/expedited-screening.vbs:66]: Moved to STAT/DISQ portion of script.

**Review by C-Love — APPROVED** [review]
Looks good.

**Pull request merged**